beanstalk_worker/Dockerfile.build_worker

```diff
@@ -0,0 +1,18 @@
+FROM registry.centos.org/centos/centos
+MAINTAINER Bamacharan Kundu<bkundu@redhat.com>
+RUN yum update -y && \
+    yum install python -y && \
+    yum clean all
+
+RUN mkdir -p /run_worker
+ADD beanstalkc.py /run_worker
+ADD ca.crt /run_worker
+ADD node.kubeconfig /run_worker
+ADD worker_start_build.py /run_worker
+RUN chmod a+x /run_worker/*
+ADD oc /usr/bin
+RUN chmod a+x /usr/bin/oc
```

Comment:
You can add all the files once, and `chmod` the two directories / files in one `RUN`